src/app.py

```diff
@@ -28,16 +34,44 @@
                     options=[{'label': "Rating", 'value': "averageRating"},
                             {"label": "Runtime", "value": "runtimeMinutes"}])
             ])
-        ])
+        ]),
+    #boxplot
+    dbc.Row([
+        dbc.Col([
+            html.Iframe(
+                id='box',
+                style={'width': '100%', 'height': '400px'})
+            ]),
+            html.Div([
+                html.H6("Region(color) for boxplot"),
+                dcc.Dropdown(
+                    id='region', value = 'US',
+                    options=[{'label': i, 'value': i} for i in region_list],
+                    multi=True
+                )
+            ])
     ])
+])
 
-
+# callback for line plot
 @app.callback(
     Output('line', 'srcDoc'),
     Input('ycol', 'value'))
+
```

Comment:
I think the convention is to not have a blank line after the function decorator, could you remove it?